CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## TBD
```

Comment:
Just to check - this will be a bug fix, so patch release rather than minor release?

Reply:
Yep